docs/relnotes.md

```diff
@@ -0,0 +1,122 @@
+---
+hide:
+  - navigation
+---
+
+# Release Notes
+
+This is the list of changes to Taipy releases as they were published.
+
+## Community edition: 1.1
+
+Published on 2022-06.
+
+[`taipy` 1.1](https://pypi.org/project/taipy/1.1.0/) contains the latest
+[`taipy-gui` 1.1](https://pypi.org/project/taipy-gui/1.1.0/), 
+[`taipy-core` 1.1](https://pypi.org/project/taipy-core/1.1.0/) and
+[`taipy-rest` 1.1](https://pypi.org/project/taipy-rest/1.1.0/) packages.
+
+
+#### Improvements
+
+**`taipy-gui`**<br/>1.1.0
+
+   - The `State^` instance can be initialized in a user-defined function. See the _on_init_
+     attribute of the `Gui^` class for more details.
```

Comment:
attribute => method ?
can't we write Gui.on_init^ ?

Reply:
Unfortunately no.
_on_init()_  has no specific entry where an anchor would be created.

We simply document this attribute in the body of the class description.